src/bloom-filter.ts

```diff
@@ -60,4 +60,21 @@ export default class BloomFilter extends BaseFilter {
   public mexists(items: any[]): Promise<number[]> {
     return this.client.call('BF.MEXISTS', this.name, ...items)
   }
+  
+  
+   /**
+   * Get filter info
+   * @return Array of string
+   */
+  public info(): Promise<string[]> {
+    return this.client.call('BF.INFO', this.name)
+  }
+  
+   /**
+   * delete a filter
+   * @return OK
+   */
+  public del(): Promise<number[]> {
+    return this.client.call('DEL', this.name)
+  }
```

Comment:
You should know that there is `BaseFilter.reset()` method.

Reply:
You are right. The function name reset mislead me, and I just ignore the function content.